templates/desktop/government.html

```diff
@@ -1,91 +1,97 @@
-                    <p><span>&ldquo;</span>We realised that Windows would cost us &euro;2 million more than Ubuntu every year.<span>&rdquo;</span></p>
-                </blockquote>
-                <p><a href="https://insights.ubuntu.com/2010/11/10/la-gendarmerie-nationale-upgrades-85000-pcs-to-ubuntu-desktop-edition/" class="external">Read the <span class="accessibility-aid">'La Gendarmerie Nationale, France' </span>case study</a></p>
-            </div>
-            <div class="four-col">
-                <h3>City of Munich, Germany</h3>
-                <blockquote class="pull-quote">
-                    <p><span>&ldquo;</span>Ubuntu satisfies our requirements best by combining the low costs and freedom of open source software with ongoing support.<span>&rdquo;</span></p>
-                </blockquote>
-                <p><a href="https://insights.ubuntu.com/2014/07/07/ubuntu-and-open-source-help-the-city-of-munich-save-millions/" class="external">Read the <span class="accessibility-aid">'City of Munich, Germany' </span>case study</a></p>
-            </div>
-            <div class="four-col last-col">
-                <h3>Uttar Pradesh Government, India</h3>
-                <p class="headline">Uttar Pradesh boosted IT literacy by putting millions laptops running Ubuntu in schools.</p>
-                <p><a href="https://insights.ubuntu.com/2013/12/04/ubuntu-helps-millions-of-indian-students-to-learn/" class="external">Read the <span class="accessibility-aid">'Uttar Pradesh Government, India' </span>case study</a></p>
-            </div>
-        </div>
+<section class="p-strip--accent is-deep">
+  <div class="row">
+    <div class="col-12">
+      <h2>Used by governments across the globe</h2>
+      <img class="u-hidden--small u-align--center" src="{{ ASSET_SERVER_URL }}9a75cea1-desktop-government-map.jpg" />
+    </div>
+  </div>
+  <div class="row p-divider">
+    <div class="col-4 p-divider__block">
+      <h3>La Gendarmerie Nationale, France</h3>
+      <blockquote class="p-pull-quote--accent">
+        <p>We realised that Windows would cost us &euro;2 million more than Ubuntu every year.</p>
+      </blockquote>
+      <p><a href="https://insights.ubuntu.com/2010/11/10/la-gendarmerie-nationale-upgrades-85000-pcs-to-ubuntu-desktop-edition/" class="p-link--external">Read the <span class="accessibility-aid">'La Gendarmerie Nationale, France' </span>case study</a></p>
+    </div>
+    <div class="col-4 p-divider__block">
+      <h3>City of Munich, Germany</h3>
+      <blockquote class="p-pull-quote--accent">
+        <p>Ubuntu satisfies our requirements best by combining the low costs and freedom of open source software with ongoing support.</p>
+      </blockquote>
+      <p><a href="https://insights.ubuntu.com/2014/07/07/ubuntu-and-open-source-help-the-city-of-munich-save-millions/" class="p-link--external">Read the <span class="accessibility-aid">'City of Munich, Germany'</span> case study</a></p>
```

Comment:
Instead of `accessibility-aid` please use Vanilla's off screen utility: https://docs.vanillaframework.io/en/utilities/off-screen